telemetry breaks in GH CI

reported in [discord](https://discord.com/channels/485586884165107732/903647230655881226/1084867883512909865) by @rocco-fortuna

> Hi, I'm having issues loading a .mlem model from a cml github actions pipeline, even though the exact same code/data/model works perfectly locally (edited)

```py
Traceback (most recent call last):
  File "./programming-language-classification/run_eval.py", line 16, in <module>
    mlem_model = load(model_path)
  File "/usr/local/lib/python3.8/dist-packages/mlem/core/metadata.py", line 156, in load
    meta = load_meta(
  File "/usr/local/lib/python3.8/dist-packages/mlem/telemetry.py", line 56, in inner
    telemetry.send_event(
  File "/usr/local/lib/python3.8/dist-packages/iterative_telemetry/__init__.py", line 125, in send_event
    self.send(
  File "/usr/local/lib/python3.8/dist-packages/iterative_telemetry/__init__.py", line 151, in send
    payload.update(self._runtime_info())
  File "/usr/local/lib/python3.8/dist-packages/iterative_telemetry/__init__.py", line 213, in _runtime_info
    ci_id = _generate_ci_id()
  File "/usr/local/lib/python3.8/dist-packages/iterative_telemetry/__init__.py", line 335, in _generate_ci_id
    res = generator()
  File "/usr/local/lib/python3.8/dist-packages/iterative_telemetry/__init__.py", line 286, in _generate_github_id
    user_id = subprocess.check_output(  # nosec B603, B607
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 493, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'gh'
Error: Process completed with exit code 1.
```

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -31,7 +31,7 @@ repos:
     hooks:
       - id: pyupgrade
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
   - repo: https://github.com/pycqa/flake8

--- a/src/iterative_telemetry/__init__.py
+++ b/src/iterative_telemetry/__init__.py
@@ -287,7 +287,7 @@ def _generate_github_id():
             ["gh", "api", f"users/{actor}", "--jq", ".name, .login, .id"],
             text=True,
         )
-    except subprocess.SubprocessError:
+    except (subprocess.SubprocessError, FileNotFoundError):
         return None
     return group_id, user_id
 

